src/main/java/com/car/app/resource/CarResource.java

```diff
@@ -17,8 +17,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
```

Comment:
The resources classes should be placed in a packaged called boundary.
Could you please review it ?